dev/devicelab/README.md

```diff
@@ -211,31 +203,36 @@ A task runs in its own standalone Dart VM and reports results via Dart VM
 service protocol. This ensures that tasks do not interfere with each other and
 lets the CI system time out and clean up tasks that get stuck.
 
-## Adding tests to the CI environment
+## Adding tests to continuous integration
 
-The `manifest.yaml` file describes a subset of tests we run in the CI. To add
-your test edit `manifest.yaml` and add the following in the "tasks" dictionary:
-
-```
-  {NAME_OF_TEST}:
-    description: {DESCRIPTION}
-    stage: {STAGE}
-    required_agent_capabilities: {CAPABILITIES}
-```
+<a name="adding-tests-to-ci"/>
 
-Where:
+There are several PRs needed to add a DeviceLab task to CI.
 
-- `{NAME_OF_TEST}` is the name of your test that also matches the name of the
+_TASK_- the name of your test that also matches the name of the
   file in `bin/tasks` without the `.dart` extension.
-- `{DESCRIPTION}` is the plain English description of your test that helps
-  others understand what this test is testing.
-- `{STAGE}` is `devicelab` if you want to run on Android, or `devicelab_ios` if
-  you want to run on iOS.
-- `{CAPABILITIES}` is an array that lists the capabilities required of
-  the test agent (the computer that runs the test) to run your test. As of writing,
-  the available capabilities are: `linux`, `linux/android`, `linux-vm`,
-  `mac`, `mac/ios`, `mac/iphonexs`, `mac/ios32`, `mac-catalina/ios`,
-  `mac-catalina/android`, `ios/gl-render-image`, `windows`, `windows/android`.
+
+1. Add prod builder to [flutter/infra devicelab_config.star](https://github.com/flutter/infra/blob/master/config/devicelab_config.star)
+  - Example PR: https://github.com/flutter/infra/pull/401/files
+  - This will need to soak for 15 minutes after merged to propagate
```

Comment:
Maybe instructions for how you can tell it's ready?

Reply:
Done - Check LUCI console for devicelab tasks